invokeai/backend/stable_diffusion/image_degradation/utils_image.py

```diff
@@ -562,18 +562,14 @@ def rgb2ycbcr(img, only_y=True):
     if only_y:
         rlt = np.dot(img, [65.481, 128.553, 24.966]) / 255.0 + 16.0
     else:
-        rlt = (
-            np.matmul(
-                img,
-                [
-                    [65.481, -37.797, 112.0],
-                    [128.553, -74.203, -93.786],
-                    [24.966, 112.0, -18.214],
-                ],
-            )
-            / 255.0
-            + [16, 128, 128]
-        )
+        rlt = np.matmul(
```

Comment:
Why not just use skimage.color for this stuff? https://scikit-image.org/docs/stable/api/skimage.color.html

Reply:
This file is unused code and removed in #4433